ase/api/src/types_int.h

```diff
@@ -95,7 +95,8 @@ struct _fpga_handle {
 	void *umsg_virt;	    // umsg Virtual Memory pointer
 	uint64_t umsg_size;	    // umsg Virtual Memory Size
 	uint64_t *umsg_iova;	    // umsg IOVA from driver
-	bool fpgaMMIO_is_mapped;  // is MMIO mapped?
+	int pasid;
+	bool fpgaMMIO_is_mapped;    // is MMIO mapped?
```

Comment:
Is this being set?  I didn't notice it.

Reply:
Yes. Toward the end of ase_fpgaBindSVA().